spec/lexers/nial_spec.rb

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+describe Rouge::Lexers::Nial do
```

Comment:
**suggestion:** Would it be possible to fix up the indentation on this spec as well 🙏🏼 ?

Reply:
I've changed it to 2 spaces.